addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -45,7 +45,29 @@
 import org.zaproxy.zap.model.Vulnerabilities;
 import org.zaproxy.zap.model.Vulnerability;
 
-/** a scanner that looks for Path Traversal vulnerabilities */
+/**
+ * A scanner that looks for Path Traversal vulnerabilities (see <a
+ * href="https://owasp.org/www-community/attacks/Path_Traversal">OWASP Path Traversal</a>).
+ *
+ * <p>The following techniques are performed (depending on the selected attack strength) both with
+ * unix and windows specific paths and separators:
+ *
+ * <ul>
+ *   <li>Absolute Path Traversal
+ *   <li>Directory climbing (../)
+ *   <li>Using file URI scheme
+ *   <li>Using null byte injection
+ * </ul>
+ *
+ * <p>TODO: There are still techniques to add, e.g.:
+ *
+ * <ul>
+ *   <li>Encoding (%2e%2e%2f - represents ../)
+ *   <li>Double encoding (%252e%252e%252f - represents ../)
+ *   <li>UTF-8 encoding (..%c0%af - represents ../)
+ *   <li>Nested traversal sequences (e.g. ....//)
+ * </ul>
```

Comment:
I know some rules have TODOs, however, I think we'd rather see this as a new enhancement issue in the main repo: https://github.com/zaproxy/zaproxy . More likely to be tackled there.

Also it's likely that more tests/payloads can only happen at INSANE due to expectations/guidelines around # of attempts etc. (Though I haven't dug specifically and checked.)

Reply:
enhancement request sounds good and yes, according to current recommendations (found in ActiveScannerTestUtils), additional checks can only be added at INSANE

fixed